cmake/partition_manager.cmake

```diff
@@ -185,6 +185,19 @@ if (CONFIG_PM_EXTERNAL_FLASH)
     )
 endif()
 
+if (CONFIG_NRF53_UPGRADE_NETWORK_CORE
+  AND CONFIG_HCI_RPMSG_BUILD_STRATEGY_FROM_SOURCE
+  AND (mcuboot_UPDATEABLE_IMAGE_NUMBER GREATER 1))
+  add_region(
+    NAME ram_flash
+    SIZE 0x40000
```

Comment:
These values should ideally come from the DT node of flash_ctrl if possible.

Reply:
You already have a kconfig symbol for the FLASH size that you can use `CONFIG_FLASH_SIZE` from `zephyr/arch/kconfig`

Reply:
CONFIG_FLASH_SIZE will be wrong it needs to be CPUNET_FLASH_SIZE